### ISSUE TITLE
nodiratime causes linux nfs mount fails with "Function not implemented", which may confuse users.

### ISSUE THREAD

**Issue by zxvv** [joyent/smartos-live#685]
In an LX zone, using the "nodiratime" option causes the linux mount utility to fail:

```shell
mount -o noatime,nodiratime,soft,intr,ac server:/a /a
mount.nfs: Function not implemented
```

Some users may not be aware that a simple fix for this is to remove the nodiratime option.
Most of the other options work (eg, noatime, ac, async, etc.), and as far as I can tell this is the one option that it might be helpful to mention.

I personally would find it very helpful if it were documented somewhere.

If it were mentioned on the wiki, say, I would have found it fairly quickly, if for instance in:
https://wiki.smartos.org/display/DOC/Configuring+NFS+in+SmartOS

**Comment by jjelinek** [comment]
Thanks, I opened bug https://smartos.org/bugview/OS-5820 for this.

**Comment by zxvv** [comment]
Thanks for considering this!

**Comment by jjelinek** [comment]
Fixed in commit ee2dc14
